cmd/manager/main.go

```diff
@@ -162,7 +162,13 @@ func main() {
 	}
 
 	// Grab platform status to determine where OpenShift is installed
```

Comment:
nit: fix comment to specify that this is both the infra status and the platform status